exodus_core/analysis/static_analysis.py

```diff
@@ -16,8 +16,8 @@
 import subprocess
 import tempfile
 import time
-import urllib.request
```

Comment:
I think we should remove urllib from the requirements.txt file too if it is not used anywhere else.